hunspell-dict.yaml

```diff
@@ -0,0 +1,162 @@
+      - hunspell-dictionary-ckb
+      - hunspell-dictionary-cs
+      - hunspell-dictionary-da
+      - hunspell-dictionary-de
+      - hunspell-dictionary-el
+      - hunspell-dictionary-en
+      - hunspell-dictionary-eo
+      - hunspell-dictionary-es
+      - hunspell-dictionary-et
+      - hunspell-dictionary-fr
+      - hunspell-dictionary-gd
+      - hunspell-dictionary-gl
+      - hunspell-dictionary-gu
+      - hunspell-dictionary-gug
+      - hunspell-dictionary-he
+      - hunspell-dictionary-hi
+      - hunspell-dictionary-hr
+      - hunspell-dictionary-hu
+      - hunspell-dictionary-id
+      - hunspell-dictionary-is
+      - hunspell-dictionary-it
+      - hunspell-dictionary-kmr
+      - hunspell-dictionary-ko
+      - hunspell-dictionary-lo
+      - hunspell-dictionary-lt
+      - hunspell-dictionary-lv
+      - hunspell-dictionary-mn
+      - hunspell-dictionary-nb
+      - hunspell-dictionary-ne
+      - hunspell-dictionary-nl
+      - hunspell-dictionary-nn
+      - hunspell-dictionary-oc
+      - hunspell-dictionary-pl
+      - hunspell-dictionary-pt
+      - hunspell-dictionary-ro
+      - hunspell-dictionary-ru
+      - hunspell-dictionary-si
+      - hunspell-dictionary-sk
+      - hunspell-dictionary-sl
+      - hunspell-dictionary-sq
+      - hunspell-dictionary-sr
+      - hunspell-dictionary-sv
+      - hunspell-dictionary-sw
+      - hunspell-dictionary-te
+      - hunspell-dictionary-th
+      - hunspell-dictionary-tr
+      - hunspell-dictionary-uk
+      - hunspell-dictionary-vi
+
+environment:
+  contents:
+    packages:
+      - autoconf
+      - automake
+      - build-base
+      - busybox
+      - ca-certificates-bundle
+      - gettext-dev
+      - git
+      - ncurses-dev
+      - readline-dev
+
+pipeline:
+  - runs: |
+      # libreoffice git repos don't provide stable download URLs, and this project doesn't use tags for versioning.
+      # our git-checkout pipeline requires a tag to checkout, so we need to do it this way...
+      git clone  https://anongit.freedesktop.org/git/libreoffice/dictionaries.git
+      cd dictionaries
+
+      # Bump this commit when updating
+      git checkout fc2dc383a40e56c0118866c9110954bebc6ea41d
+
+data:
+  - name: _dictionaries
+    items:
+      af:
+      an:
+      ar:
+      bg:
+      bn:
+      bo:
+      br:
+      bs:
+      ca:
+      ckb:
+      cs:
+      da:
+      de:
+      el:
+      en:
+      eo:
+      es:
+      et:
+      fr:
+      gd:
+      gl:
+      gu:
+      gug:
+      he:
+      hi:
+      hr:
+      hu:
+      id:
+      is:
+      it:
+      kmr:
+      ko:
+      lo:
+      lt:
+      lv:
+      mn:
+      nb:
+      ne:
+      nl:
+      nn:
+      oc:
+      pl:
+      pt:
+      ro:
+      ru:
+      si:
+      sk:
+      sl:
+      sq:
+      sr:
+      sv:
+      sw:
+      te:
+      th:
+      tr:
+      uk:
+      vi:
+
+subpackages:
+  - range: _dictionaries
+    name: hunspell-dictionary-${{range.key}}
```

Comment:
Just curious - some other distros seem to call these packages "hunspell-$lang" without the -dictionary- part, did you copy that convention from somewhere?

Reply:
usually `-lang` is for locales (like `glibc-lang-foo`) these are dictionaries, so I didn't want to make it confusing

on alpine they just use `hunspell-en-...` without `-dictionaries` but I thought having a separate package like this enables us also to have an `-all` package that depends on the subpackages

This enables apk to properly handle adding `-all` after a subpackage without overwriting files